tea.go

```diff
@@ -562,7 +562,7 @@ func (p *Program) StartReturningModel() (Model, error) {
 	}
 }
 
-// Start initializes the program. Ignores the final model.
+// Start initializes the program and waits for it to end.
```

Comment:
I would emphasize that it does more than just initializing the program: it executes its event loop and waits for the program to be terminated by a `tea.Quit` message.

Reply:
Makes sense. Let me fire up my editor.

Reply:
How about now?

Reply:
Keep in mind that the program can also be terminated by `Program.Quit`, `SIGINT` and `SIGTERM`. On that note I might remove the `tea.Quit` detail and simply state that the method blocks.

Reply:
Technically `Program.Quit` is just `tea.Quit` as a function, but you're right. Instead of removing it, I think pointing that out in the docs wouldn't be a bad idea either. `tea` setting up its own signal handler isn't exactly obvious and users should be made aware of it.

Reply:
That's a great point, especially now that GoDocs support linking.